src/server/api/user.js

```diff
@@ -211,23 +211,29 @@ export const resolvers = {
       }
       return formatPage(query, { after, first });
     },
-    organizations: async (user, { role }) => {
+    organizations: async (user, { role, active }) => {
```

Comment:
I think this value should be false by default, rather than true, and places that need it to be true, set true. So by default inactive orgs are hidden.

Reply:
ah right. my bad.